{"uid":"f7d36b0ad6f1c24b","name":"test_change_name","fullName":"tests.test_example.ExampleTestCase#test_change_name","historyId":"401c7467121e7dc775efa0719d6717a6","time":{"start":1757402270481,"stop":1757402274133,"duration":3652},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","statusTrace":"self = <test_example.ExampleTestCase testMethod=test_change_name>\n\n    def test_change_name(self):\n>       self.home.open(OLLAMA_URL)\n\ntests/test_example.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/home_page.py:21: in open\n    self.driver.get(url)\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:483: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe2fea73190>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"Reached error page: about:neterror?e=connection....sys.mjs:156:19\\\\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       UnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\nE       checkReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nE       onNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nE       emit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nE       receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":5,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase","time":{"start":1757402270481,"stop":1757402270481,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase::0","time":{"start":1757402278661,"stop":1757402278661,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_example"},{"name":"subSuite","value":"ExampleTestCase"},{"name":"host","value":"pkrvm7jw40e0xgp"},{"name":"thread","value":"2634-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_example"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"5485304c76c1b730","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1757402263566,"stop":1757402266939,"duration":3373}},{"uid":"a9b724fe1c213ecd","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x55ee565d608a <unknown>\n#1 0x55ee56075a70 <unknown>\n#2 0x55ee5606ccf1 <unknown>\n#3 0x55ee5605e099 <unknown>\n#4 0x55ee5605fde0 <unknown>\n#5 0x55ee5605e43e <unknown>\n#6 0x55ee5605ddaa <unknown>\n#7 0x55ee5605da85 <unknown>\n#8 0x55ee5605b8ee <unknown>\n#9 0x55ee5605c0ca <unknown>\n#10 0x55ee56078f99 <unknown>\n#11 0x55ee56113d65 <unknown>\n#12 0x55ee560ed3e2 <unknown>\n#13 0x55ee5611314f <unknown>\n#14 0x55ee560ed1b3 <unknown>\n#15 0x55ee560b959b <unknown>\n#16 0x55ee560ba971 <unknown>\n#17 0x55ee5659b25b <unknown>\n#18 0x55ee5659efa9 <unknown>\n#19 0x55ee56582339 <unknown>\n#20 0x55ee5659fb58 <unknown>\n#21 0x55ee56566c1f <unknown>\n#22 0x55ee565c3118 <unknown>\n#23 0x55ee565c32f6 <unknown>\n#24 0x55ee565d5066 <unknown>\n#25 0x7f8455e9caa4 <unknown>\n#26 0x7f8455f29c3c <unknown>","time":{"start":1757402261385,"stop":1757402269032,"duration":7647}},{"uid":"55d89dacf98da8ad","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1757402258625,"stop":1757402262031,"duration":3406}},{"uid":"4cdbbf8f262cfc72","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x55bc71a1e08a <unknown>\n#1 0x55bc714bda70 <unknown>\n#2 0x55bc714b4cf1 <unknown>\n#3 0x55bc714a6099 <unknown>\n#4 0x55bc714a7de0 <unknown>\n#5 0x55bc714a643e <unknown>\n#6 0x55bc714a5daa <unknown>\n#7 0x55bc714a5a85 <unknown>\n#8 0x55bc714a38ee <unknown>\n#9 0x55bc714a40ca <unknown>\n#10 0x55bc714c0f99 <unknown>\n#11 0x55bc7155bd65 <unknown>\n#12 0x55bc715353e2 <unknown>\n#13 0x55bc7155b14f <unknown>\n#14 0x55bc715351b3 <unknown>\n#15 0x55bc7150159b <unknown>\n#16 0x55bc71502971 <unknown>\n#17 0x55bc719e325b <unknown>\n#18 0x55bc719e6fa9 <unknown>\n#19 0x55bc719ca339 <unknown>\n#20 0x55bc719e7b58 <unknown>\n#21 0x55bc719aec1f <unknown>\n#22 0x55bc71a0b118 <unknown>\n#23 0x55bc71a0b2f6 <unknown>\n#24 0x55bc71a1d066 <unknown>\n#25 0x7fbc3409caa4 <unknown>\n#26 0x7fbc34129c3c <unknown>","time":{"start":1757402257150,"stop":1757402266970,"duration":9820}},{"uid":"49ab3f063cec00d3","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x563d8bd5d08a <unknown>\n#1 0x563d8b7fca70 <unknown>\n#2 0x563d8b7f3cf1 <unknown>\n#3 0x563d8b7e5099 <unknown>\n#4 0x563d8b7e6de0 <unknown>\n#5 0x563d8b7e543e <unknown>\n#6 0x563d8b7e4daa <unknown>\n#7 0x563d8b7e4a85 <unknown>\n#8 0x563d8b7e28ee <unknown>\n#9 0x563d8b7e30ca <unknown>\n#10 0x563d8b7fff99 <unknown>\n#11 0x563d8b89ad65 <unknown>\n#12 0x563d8b8743e2 <unknown>\n#13 0x563d8b89a14f <unknown>\n#14 0x563d8b8741b3 <unknown>\n#15 0x563d8b84059b <unknown>\n#16 0x563d8b841971 <unknown>\n#17 0x563d8bd2225b <unknown>\n#18 0x563d8bd25fa9 <unknown>\n#19 0x563d8bd09339 <unknown>\n#20 0x563d8bd26b58 <unknown>\n#21 0x563d8bcedc1f <unknown>\n#22 0x563d8bd4a118 <unknown>\n#23 0x563d8bd4a2f6 <unknown>\n#24 0x563d8bd5c066 <unknown>\n#25 0x7ffb8709caa4 <unknown>\n#26 0x7ffb87129c3c <unknown>","time":{"start":1757402254563,"stop":1757402262287,"duration":7724}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"f7d36b0ad6f1c24b.json","parameterValues":[]}